packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -42,19 +42,32 @@ def CallDhclient(
     logger.warning('Could not enable interfaces %s.', interfaces)
 
 
-def CallDhclientIpv6(interfaces, logger, dhclient_script=None):
+def CallDhclientIpv6(interfaces, logger, dhclient_script=None,
+                     release_lease=False):
   """Configure the network interfaces for IPv6 using dhclient.
 
   Args:
     interface: string, the output device names for enabling IPv6.
     logger: logger object, used to write to SysLog and serial port.
     dhclient_script: string, the path to a dhclient script used by dhclient.
+    release_lease: Release the IPv6 lease.
   """
-  logger.info('Enabling IPv6 on the Ethernet interfaces %s.', interfaces)
+  logger.info('Calling Dhclient for IPv6 configuration '
+              'on the Ethernet interfaces %s.', interfaces)
 
   timeout_command = ['timeout', '5']
   dhclient_command = ['dhclient']
 
+  if release_lease:
+    try:
+      subprocess.check_call(
+          timeout_command + dhclient_command + [
+              '-1', '-6', '-r', '-v'] + interfaces)
```

Comment:
do we need these extra args to do a release?
also, everywhere else, we do `dhclient -x`, why is this one using -r? does the dhcp server need the release?

Reply:
-r is releasing the lease. -x does not matter here since there is no script to run when releasing the release. We want to release the lease and force Private Access clients to not use the IPv6 address any more.

Reply:
-r and -x kill running dhclient processes (based on pidfile, see also my comment regarding dhclient exclusivity), where -r also sends a release to the DHCP server. if you think the nanny cares to have it formally released then i don't have any concern, i just wasn't sure if it was necessary. the other arguments should not be needed, like '-1'/'run once' and '-6'/'get ipv6 lease' have no meaning in a lease release context.